Filters/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1081
+! Blocked by CNAME ||iterable.com^
+@@||links.bringatrailer.com^|
```

Comment:
In this case, better exclude blocking rule. 
Only this request may be blocked: `js.iterable.com/analytics.js` (like in our TPF)

Reply:
You can see more affected domains here: https://urlfilter.adtidy.org/v1/findCloaked?domain=iterable.com